cost/aws/elb/clb_unused/README.md

```diff
@@ -0,0 +1,50 @@
+## AWS Unused Classic Load Balancers (CLB) 
+ 
+### What it does
+This policy checks all CLB to determine if any are unused (have no healthy instances) and allows them to be deleted by the user after approval.
+
+Note:Elastic Load Balancing (ELB) supports three types of load balancers: Application Load Balancers, Network Load Balancers, and Classic Load Balancers.
+
+### Functional Details
+ 
+The policy leverages the AWS elasticloadbalancing API to determine if the CLB is in use.
+ 
+When an unused CLB is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have the option to delete the CLB after manual approval if needed.
+ 
+#### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
+- *Ignore tags* - CLB with any of these tags will be ignored 
+ 
+### Required RightScale Roles
+ 
+- policy_manager
+- admin or credential_viewer
+
+### AWS Required Permissions
+
+This policy requires permissions to describe AWS LoadBalancers,InstanceHealth, tags and DeleteLoadBalancer. 
```

Comment:
Thanks for adding this. Please add a space between `LoadBalancers,` and `InstanceHealth` and also between `tags` and `and`.

Reply:
Added a space between LoadBalancers, and InstanceHealth.
There was a space already between tags and and.